<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="P4A-508SCN9" name="Command line test" tests="3" failures="1" timestamp="25 Jul 2018 06:33:01 GMT" time="26.699" errors="0">
  <testcase name="LoginInvalid" time="0.012" classname="tests.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [Tested123] but found [Tested]">
      <![CDATA[java.lang.AssertionError: expected [Tested123] but found [Tested]
	at tests.LoginTest.LoginInvalid(LoginTest.java:21)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- LoginInvalid -->
  <testcase name="LoginValid" time="0.005" classname="tests.LoginTest"/>
  <testcase name="TestRegistration" time="7.139" classname="tests.RegistrationTest"/>
</testsuite> <!-- Command line test -->
